### PR TITLE
Turn led off

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

**Comment by luciipak** [comment]
@jperson got it, yeah I'm not sure why the text isn't changing here. I'm wondering if it's because some error is happening and it skips the function that resets the text: `[error] External video encoding failed: 0`

https://github.com/CuriBio/nautilus_acquisition/assets/84344924/1cfdc60d-d969-4db7-96eb-7017f39048b5

**Comment by jperson** [comment]
You probably don't have ffmpeg installed

**Comment by luciipak** [comment]
> You probably don't have ffmpeg installed
👍

File: src/app/src/mainwindow.h [hunk 1]
```diff
@@ -305,7 +305,7 @@ class MainWindow : public QMainWindow {
                 m_curState = (stopAcquisition_LiveViewRunning()) ? LiveViewRunning : Error;
             }},
             { {LiveViewAcquisitionRunning, PostProcessing}, [this]() {
-                m_curState = (startPostProcessing_LiveViewRunning()) ? PostProcessingLiveView : Error;
+                m_curState = (startPostProcessing_LiveViewRunning()) ? PostProcessing : Error;
```
Comment (luciipak) [inline src/app/src/mainwindow.h:308]: if this remains `PostProcessingLiveView `, then `postProcessingDone_LiveViewRunning` gets called. This change calls `postProcessingDone`

File: resources/plate_formats/CuriBio 24 Well Plate.toml [hunk 2]
```diff
@@ -8,152 +8,11 @@ location = [
...
-is_active = true
-grid_position = { row = 3, col = 5 }
-center_coordinates = { x_pos = 2788.0, y_pos = 1769.0 }
+num_wells = 24
+roiSize = 128
+numWellsH = 2
+numWellsV = 2
+wellSpacing = 18000
+hOffset = 0
+vOffset = 0
```
Comment (luciipak) [inline resources/plate_formats/CuriBio 24 Well Plate.toml:17]: do these file changes need to be in the changelog?

File: src/app/src/mainwindow.h (hunk 1 above)
Comment (luciipak) [reply src/app/src/mainwindow.h:308]: should I remove `postProcessingDone_LiveViewRunning` and `PostProcessingLiveView`? or leave them for now?

File: resources/plate_formats/CuriBio 24 Well Plate.toml (hunk 2 above)
Comment (jperson) [reply resources/plate_formats/CuriBio 24 Well Plate.toml:17]: doesn't hurt to add a line saying the platemaps were updated

File: src/app/src/mainwindow.h (hunk 1 above)
Comment (jperson) [reply src/app/src/mainwindow.h:308]: I'd leave it for now in case they change their minds about it

**Review by jperson — APPROVED** [review]

**Pull request merged**